core/test/libraries/Volatility.t.sol

```diff
@@ -79,40 +96,13 @@ contract VolatilityTest is Test {
             Volatility.FeeGrowthGlobals(
                 1501955347902231987349614320458936,
                 527278396421895291380335427321388844898052,
-                uint32(block.timestamp)
+                uint32(1660599905)
```

Comment:
Why? What's the significance of this timestamp?

Reply:
Just a random one :)